Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmissionParameter.java

```diff
@@ -64,6 +64,7 @@
     private List<String> jars = new ArrayList<>();
     private List<String> args = new ArrayList<>();
     private Map<String, Object> jobConfig = new HashMap<>();
+    private String sparkEventsRelativePath = "spark-events";
```

Comment:
Please revert change of this file. Class `SparkSubmissionParameter` is designed to prepare parameters for HDInsight spark batch job. Actually we have a class `CreateSparkBatchJobParameters`, which is designed for Cosmos serverless spark batch job.